pkg/apis/crd/v1alpha2/types.go

```diff
@@ -306,28 +306,40 @@ type IPPoolSpec struct {
 	IPRanges []SubnetIPRange `json:"ipRanges"`
 }
 
-// SubnetIPRange is a set of contiguous IP addresses, represented by a CIDR or a pair of start and end IPs,
-// along with subnet definition.
-type SubnetIPRange struct {
-	IPRange `json:",inline"`
+// SubnetSpec is subnet specification for IP Range
+type SubnetSpec struct {
 	// Gateway IP for this subnet, eg. 10.10.1.1
 	Gateway string `json:"gateway"`
 	// Prefix length for the subnet, eg. 24
-	PrefixLen string `json:"prefixLen"`
+	PrefixLength int32 `json:"prefixLength"`
```

Comment:
pedant me says we could use 'int8' , but it's not really important!

Reply:
I don't see any other int8-s or even int16-s defined in types, so I think its safer to stick to int32 (it is also consistent with PrefixLength in other object)

Reply:
That's ok, my memory must be failing me.